src/main/java/WayofTime/bloodmagic/ritual/IMasterRitualStone.java

```diff
@@ -55,7 +55,7 @@
 
     void setActiveWillConfig(EntityPlayer player, List<EnumDemonWillType> typeList);
 
-    boolean setBlockRangeByBounds(EntityPlayer player, String range, BlockPos offset1, BlockPos offset2);
+    int setBlockRangeByBounds(EntityPlayer player, String range, BlockPos offset1, BlockPos offset2);
```

Comment:
Add a javadoc explaining what this int should be. However, it would probably be better to use an enum since those are fairly self-documenting.